news/2024/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks.md

```diff
@@ -0,0 +1,308 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+Hmm I feel there are many moments I could say, but let me think... when I reached 25,000, it's exactly when I started to REALLY believe I could reach 50k, since I was halfway through, so it's very memorable for me. And of course, when I was reaching 50,000 and 100,000 SS. When I was going for 50k, it was very tough, because I didn't really planned very well the final maps, and was needing to search for anything I could SS more easily. This made me get very exhausted in that day. Due that, for 100k SS I prepared much better, saving many short Normal diffs to play in the final 1000 SS, and that made the final section much faster and less exhaustive!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+How did you feel after finally reaching this goal? Did you maybe celebrate it in any particular way?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/4568537" style="background-image: url('/wiki/shared/news/2024-10-21-the-followpoint-predominador-the-grandmaster-of-ss-ranks/avatar-Predominador.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/4568537">Predominador</a></p>
+
+It was a mix of emotions, but I felt specially really relieved, because at that point I was postponing every other thing I wanted to do to avoid making it longer than it was already being (I also procrastinated a lot). Now I can continue watching Frieren \õ/ (for example)!
+
+About celebration, kinda. It didn't happen the way I planned, though. I was planning to reach it while livestreaming in a Discord call with some friends (and listening to them and chatting by text), which is something I even started to do, but the grind took longer than I expected. When I was getting closer, it was very late night already. And I didn't want to let it to the "next day" (it was already September 25th) because, fun fact, it was also my birthday (and this really wasn't planned, in my expectations I was supposed to reach it at most around day 10 of September)! So, even my friends said to finish it, then when I wake up again I could just rest and celebrate! They followed me in the most part of the journey, though. Not just in that day, some of them follow my progress since years, and I am very grateful for that :D
```

Comment:
```suggestion
About celebration, kinda. It didn't happen the way I planned, though. I was planning to reach it while livestreaming in a Discord call with some friends (and listening to them and chatting by text), which is something I even started to do, but the grind took longer than I expected.

When I was getting closer, it was very late night already. And I didn't want to let it to the "next day" (it was already September 25th) because, fun fact, it was also my birthday (and this really wasn't planned, in my expectations I was supposed to reach it at most around day 10 of September)! So, even my friends said to finish it, then when I wake up again I could just rest and celebrate! They followed me in the most part of the journey, though. Not just in that day, some of them follow my progress since years, and I am very grateful for that :D
```